.github/workflows/element-with-modules.yml

```diff
@@ -0,0 +1,351 @@
+          IS_REF: ${{ ! startsWith(matrix.opendesk, '@nordeck') }}
+        run: |
+          VERSION="${{ env.VERSION }}"
+          HUMAN_OPENDESK="v$(echo ${{ env.VERSION }} | tr "@" "\n" | tail -1)"
+          if ${{ env.IS_REF }}
+          then
+            VERSION="file:nordeck-element-web-opendesk-module.tgz"
+            HUMAN_OPENDESK="$(git describe --exclude='*' --always)"
+          fi
+          echo "  - '$VERSION'" >> ../build_config.yaml
+          HUMAN_OPENDESK="+od-${HUMAN_OPENDESK}"
+          echo "HUMAN_OPENDESK=$HUMAN_OPENDESK" >> $GITHUB_ENV
+
+      - name: Checkout Widget Lifecycle module version
+        if: ${{ matrix.lifecycle != 'none' }}
+        uses: actions/checkout@v4
+        with:
+          ref: '${{ matrix.lifecycle }}'
+          fetch-tags: true
+          fetch-depth: 0
+
+      - name: Build Widget Lifecycle module
+        if: ${{ ! (startsWith(matrix.lifecycle, '@nordeck') || startsWith(matrix.lifecycle, 'none')) }}
+        run: |
+          yarn install --frozen-lockfile
+          yarn workspace @nordeck/element-web-widget-lifecycle-module package
+          mv packages/element-web-widget-lifecycle-module/nordeck-element-web-widget-lifecycle-module.tgz ../
+
+      - name: Generate Widget Lifecycle module build_config.yaml entry
+        if: ${{ matrix.lifecycle != 'none' }}
+        env:
+          VERSION: ${{ matrix.lifecycle }}
+          IS_REF: ${{ ! startsWith(matrix.lifecycle, '@nordeck') }}
+        run: |
+          VERSION="${{ env.VERSION }}"
+          HUMAN_LIFECYCLE="v$(echo ${{ env.VERSION }} | tr "@" "\n" | tail -1)"
+          if ${{ env.IS_REF }}
+          then
+            VERSION="file:nordeck-element-web-widget-lifecycle-module.tgz"
+            HUMAN_LIFECYCLE="$(git describe --exclude='*' --always)"
+          fi
+          echo "  - '$VERSION'" >> ../build_config.yaml
+          HUMAN_LIFECYCLE="+wl-${HUMAN_LIFECYCLE}"
+          echo "HUMAN_LIFECYCLE=$HUMAN_LIFECYCLE" >> $GITHUB_ENV
+
+      - name: Checkout Widget Toggles module version
+        if: ${{ matrix.toggles != 'none' }}
+        uses: actions/checkout@v4
+        with:
+          ref: '${{ matrix.toggles }}'
+          fetch-tags: true
+          fetch-depth: 0
+
+      - name: Build Widget Toggles module
+        if: ${{ ! (startsWith(matrix.toggles, '@nordeck') || startsWith(matrix.toggles, 'none')) }}
+        run: |
+          yarn install --frozen-lockfile
+          yarn workspace @nordeck/element-web-widget-toggles-module package
+          mv packages/element-web-widget-toggles-module/nordeck-element-web-widget-toggles-module.tgz ../
+
+      - name: Generate Widget Toggles module build_config.yaml entry
+        if: ${{ matrix.toggles != 'none' }}
+        env:
+          VERSION: ${{ matrix.toggles }}
+          IS_REF: ${{ ! startsWith(matrix.toggles, '@nordeck') }}
+        run: |
+          VERSION=${{ env.VERSION }}
+          HUMAN_TOGGLES="v$(echo ${{ env.VERSION }} | tr "@" "\n" | tail -1)"
```

Comment:
This gets the version after the `@`, right? While `tr | tail` works a more intuitive way would be `cut --delimiter=@ --fields=2`.

Reply:
```
$ git describe --match="@nordeck/element-web-guest-module*" --abbrev=0 | cut --delimiter=@ --fields=2
nordeck/element-web-guest-module
```

:)

I think therefore it is more stable with `tail`